README.md

```diff
@@ -205,5 +205,47 @@ with open('../config.yaml', 'w') as file:
     yaml.dump(config, file, default_flow_style=False)
 ```
 
+# Speedy User Start
```

Comment:
Can you please follow the same header formatting for this section as the rest of the readme file? I.e. titles should be ## not #. Also please elaborate on all the features you have added in detail and include graphics using the same style as other sections. You may place any pictures in the graphics subfolder. Try to enumerate all the features as well so that it follows the same logical flow as the rest of the readme file.